example/demo-gen/demo/ICU4XFixedDecimalFormatter.mjs

```diff
@@ -0,0 +1,50 @@
+import { ICU4XDataProvider } from "../ICU4XDataProvider.mjs"
+import { ICU4XFixedDecimal } from "../ICU4XFixedDecimal.mjs"
+import { ICU4XFixedDecimalFormatter } from "../ICU4XFixedDecimalFormatter.mjs"
+import { ICU4XFixedDecimalFormatterOptions } from "../ICU4XFixedDecimalFormatterOptions.mjs"
+import { ICU4XLocale } from "../ICU4XLocale.mjs"
+
+export function formatWrite(name, grouping_strategy, some_other_config, v) {
+	var terminusArgs = arguments;
+	return (function (...args) { return this.formatWrite(...args) }).apply(
+        ICU4XFixedDecimalFormatter.tryNew.apply(
+        null,
+        [
+            ICU4XLocale.new_.apply(
+                null,
+                [
+                    terminusArgs[0]
+                ]
+            ),
+            ICU4XDataProvider.newStatic.apply(
+                null,
+                [
+                ]
+            ),
+            (function (...args) {
+            	let out = new ICU4XFixedDecimalFormatterOptions();
+            	
+            	out.groupingStrategy = args[0];
+            	
+            	out.someOtherConfig = args[1];
+            	
+            	return out;
+            }).apply(
+                null,
+                [
+                    terminusArgs[1],
+                    terminusArgs[2]
+                ]
+            )
```

Comment:
Suggestion, if easy:

```suggestion
            (function () {
            	let out = new ICU4XFixedDecimalFormatterOptions();
            	out.groupingStrategy = terminusArgs[1];
            	out.someOtherConfig = terminusArgs[2];
            	return out;
            })()
```

Reply:
Can't find an easy fix for this, the template for this function currently doesn't know what arguments it's being passed.

I think the fix would be a re-design of `ParamInfo`, so I've added it to the list of issues under this PR.